source/g4collimation/CollimationMaterials.cpp

```diff
@@ -42,6 +43,8 @@ CollimationMaterials::CollimationMaterials()
 	AddMaterial("Mo", Mo);
 	AddMaterial("W", W);
 	AddMaterial("Pb", Pb);
+	AddMaterial("Si", Si);
+	AddMaterial("SI", Si);
```

Comment:
why twice? I would leave only `Si`. In case the string can be input by the user, I would consider string comparison where the code lowers everything...

Reply:
on my ever growing to do list